integration_tests/e2e/refer.cy.ts

```diff
@@ -301,6 +302,77 @@ context('Refer', () => {
     taskListPage.shouldHaveConfirmedOasys()
   })
 
+  it('Shows the reason form page', () => {
```

Comment:
One thing I missed last night: we'll need a corresponding test in `e2eReferDisabled`